cli/azd/cmd/pipeline.go

```diff
@@ -89,7 +89,7 @@ func newPipelineConfigFlags(cmd *cobra.Command, global *internal.GlobalCommandOp
 func newPipelineConfigCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "config",
-		Short: "Create and configure your deployment pipeline by using GitHub Actions or Azure Pipelines.",
+		Short: i18nGetText(i18nCmdPipelineConfigShort),
 		Long: `Create and configure your deployment pipeline by using GitHub Actions or Azure Pipelines.
 
 For more information, go to https://aka.ms/azure-dev/pipeline.`,
```

Comment:
Missed this text? I'm assuming at this point we're putting all our command descriptions in the yaml?